src/components/Navbar.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- Hey, currently this eslint disable does not have any effect on your code, please, let's kindly remove it to keep the codebase clean and maintainable.

Reply:
### README LIVE DEMO LINK


- Hey, since we've not deployed the app and its currently broken, please, let's remove the live demo link and also, remove its navigation from the readme to keep our readme more professional.

![Screenshot from 2023-02-24 05-58-05](https://user-images.githubusercontent.com/59397159/221096424-af6a83bc-d43a-4db4-813d-ead19cd411bd.png)
![Screenshot from 2023-02-24 05-57-32](https://user-images.githubusercontent.com/59397159/221096431-e3ef93ae-4a45-42ef-8dcc-8bb22f7da43d.png)